spec/support/suspenders.rb

```diff
@@ -38,7 +38,7 @@ def rails_new
       add_fakes_to_path
 
       with_revision_for_honeybadger do
-        debug `#{system_rails_bin} _#{rails_version}_ new #{APP_NAME} -m #{rails_template_path}`
+        debug `#{system_rails_bin} _#{rails_version}_ new #{APP_NAME} --skip-spring -d postgresql -m #{rails_template_path}`
```

Comment:
Interesting that we're forcing postgres -- I had hoped it would come from the template. Going to investigate whether we need that right now but let me know if you did that intentionally.

Reply:
Not sure, I think this was part of the changes I merged in from one of your earlier commits in the PR. It might not be necessary.

Reply:
Confirmed:

```
LoadError: Error loading the 'postgresql' Active Record adapter. Missing a gem it depends on? pg is not part of the bundle. Add it to your Gemfile.
```